project-reports/2023/2023-Q1-Hyperledger-AnonCreds.md

```diff
@@ -0,0 +1,90 @@
+
+Response to the extraction of the AnonCreds capabilities from Indy and their use with other ledgers
+has been generally positive, including an uptick in attendance and activity on the AnonCreds specification.
+
+## Questions/Issues for the TSC
+
+## Releases
+
+- [anoncreds-rs] -- none
+
+## Overall Activity in the Past Quarter
+
+The Rust implementation of AnonCreds was created by duplicating the repo of its previous home in the [indy-shared-rs](https://github.com/hyperledger/indy-shared-rs) repository, and the subsequent removal of the "non-AnonCreds" parts
+of the repo. The focus of the maintainers and contributors is aligning the implementation with the "ledger-agnostic"
+elements of the specification, and updating the specification to align with the implementation. The volume of changes
+to the implementation to be ledger-agnostic has been surprisingly small. Only one Indy-specific data element needed to
+be removed from the AnonCreds API, and most of the changes have been to clean variable names to align their name to their meaning. The most significant single change was in the writing and reading of Revocation Registry state.
+
+In addition to developing the Rust implementations, wrappers for JavaScript and Python have been created/evolved and GitHub Actions have been created to automated builds and release publishing.
+
+Documentation updates in the Rust implementation repo needs attention.
+
+On November 28, 2022, we had a presentation from Mike Lodder who has been involved with AnonCreds since the early
+days of the Indy implementation. He provided a proposal for how to evolve AnonCreds from the current v1.0 specification to a v2.0 that retains (and extends in some places) the key privacy preserving features of AnonCreds with updated cryptographic primitives. The meeting agenda, notes and recordings are available [here](https://wiki.hyperledger.org/display/ANONCREDS/2022-11-28+AnonCreds+Specification+Working+Group+Meeting). Definitely worth listening to the presentation!
+
+There is no AnonCreds Quarterly Activity report available on the LFX Insights page. We will update this report to include the report when it becomes available.
+
+[AnonCreds Quarterly Activity Dashboard]: https://insights.lfx.linuxfoundation.org/projects/hyperledger%2Fanoncreds/dashboard;subTab=technical?time=%7B%22from%22:%222022-10-01T07:00:00.000Z%22,%22type%22:%22absolute%22,%22to%22:%222022-12-31T07:00:00.000Z%22%7D
+
+## Current Plans
+
+Work on the first release is nearing completion -- the first goal of the project. This work, combined with the efforts
+in the Hyperledger Indy project to eliminate the Indy SDK will mean the quick adoption of AnonCreds into the Aries frameworks. Once we have an initial release, we'll explore the use of AnonCreds outside of the Indy and even outside of Aries.
```

Comment:
I would say usage of AnonCreds outside of Indy is already actively being worked on. 

- There's a spec being worked on for Cardano: https://github.com/roots-id/cardano-anoncreds/blob/main/cardano-anoncred-methods.md
- We've been working this week with cheqd to get their guides up to date on the latest chagnes to the spec (https://docs.cheqd.io/identity/guides/using-on-ledger-resources-to-support-anoncreds), and development has started to implement the cheqd anoncreds binding into Aries Framework JavaScript

Reply:
Thanks, Timo — much appreciated.  Sorry I didn’t check in with you before posting, but I guess that is the point of this process :-).